### PR TITLE
Help Command + Announcement cleanup

### PR DESCRIPTION
**Pull request by j10a1n15**

### CODE REVIEW

File: src/commands/help.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+//
+// Written by J10a1n15.
+// See LICENSE for copyright and license notices.
+//
+
```
Comment (Su386yt) [inline src/commands/help.js:5]: One line instead of two /j

**Review by Su386yt — APPROVED** [review]

File: src/commands/help.js (hunk 1 above)
Comment (j10a1n15) [reply src/commands/help.js:5]: what im confused

Comment (Su386yt) [reply src/commands/help.js:5]: There's only one line between the header comment and the code

Comment (j10a1n15) [reply src/commands/help.js:5]: image
should it not be only one?

Comment (Su386yt) [reply src/commands/help.js:5]: Thought it was 2

Comment (j10a1n15) [reply src/commands/help.js:5]: idk, is there a style for it?

Comment (Su386yt) [reply src/commands/help.js:5]: Idk I just like 2

**Comment by j10a1n15** [comment]
Will merge then

**Pull request merged**